packages/rn-tester-e2e/test/screenObjects/buttonComponent.screen.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+import Utils from '../helpers/utils';
+
+class ButtonComponentScreen {
+  buttonScreenElement = Utils.platformSelect({
+    ios: '[label="Button"]',
+    android: '//android.view.ViewGroup/android.widget.TextView[@text="Button"]',
+  });
+
+  btnSubmitElement = Utils.platformSelect({
+    ios: '[label="Press to submit your application!"]',
+    android: '//android.widget.Button[@resource-id="button_default_styling"]',
```

Comment:
we need to change things to be like here https://github.com/kelset/react-native-e2e-jest-appium-webdriverio/blob/main/__tests__/basicE2E.test.js#L33